storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -4624,7 +4569,7 @@ public boolean isRemoteBlobExists(String blobKey) throws AuthorizationException,
         return true;
     }
 
-    private static final class Assoc<K, V> implements UnaryOperator<Map<K, V>> {
+    static final class Assoc<K, V> implements UnaryOperator<Map<K, V>> {
```

Comment:
Assoc and Dissoc don't seem to be used in new places, so is making them package private still necessary?

Reply:
Yes, I'll make them private again.  This was because as I was refactoring I kept the Assoc and  Dissoc until the very end when I fixed the threading.